src/LabelGroup.js

```diff
@@ -1,23 +1,28 @@
 import React from 'react'
 import styled from 'styled-components'
 import theme from './theme'
-import {COMMON} from './constants'
+import classnames from 'classnames'
+import {COMMON, get} from './constants'
 
 const transformChildren = children => {
   return React.Children.map(children, child => {
-    return React.cloneElement(child, {mr: 1, className: 'LabelItem'})
+    const classes = classnames(child.props['className'], 'LabelItem')
+    return React.cloneElement(child, {className: classes})
```

Comment:
After our discussion with @broccolini can you take the passed classname back out? 😂